### PR TITLE
Refactor jsdata and theme

### PR DESCRIPTION
**Pull request by dreadnip**
Having our jsData as an inline script is not ideal. It's a possible vulnerability and we have to[ trust that it will be present in our style package.
](https://github.com/sumocoders/FrameworkStylePackage/blob/master/src/js/Framework/Data.js#L9)
Symfony also recommends different ways to pass data from Twig to JS: https://symfony.com/doc/current/frontend/encore/server-data.html. The locale we used to pass in the jsData is already present as the lang attribute, which we can read in JS with
```js
document.getQuerySelector('html'].getAttribute('lang');
```
This only leaves the current theme and the path to the dark style CSS file. With this PR I propose we set both as data attributes on our body tag, as suggested by the Symfony docs. We can verify if they're properly set and then read their values in our style package.

PR in style package: https://github.com/sumocoders/FrameworkStylePackage/pull/65

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -24,10 +24,10 @@
     "php": "^7.4 || ^8.0",
     "gedmo/doctrine-extensions": "^3",
     "friendsofsymfony/jsrouting-bundle": "^2",
-    "tijsverkoyen/css-to-inline-styles": "^2.2",
```
Comment (tijsverkoyen) [inline composer.json:27]: Waarom is die opeens weg?

File: templates/base.html.twig [hunk 2]
```diff
@@ -2,7 +2,7 @@
   {{ include('@SumoCodersFrameworkCore/head.html.twig') }}
 {% endblock %}
 
-<body class="body-base" data-theme="{{ theme }}">
+<body class="body-base" data-theme-current="{{ theme() }}" data-theme-dark="{{ asset('build/style-dark') }}">
```
Comment (tijsverkoyen) [inline templates/base.html.twig:5]: Niet logischer om data-theme-path te  noemen?

File: composer.json (hunk 1 above)
Comment (dreadnip) [reply composer.json:27]: Is dependency van `"twig/cssinliner-extra": "^3.3",` dus we krijgen die via-via binnen. We gebruiken hem ook niet meer rechtstreeks, maar via die Twig extensie.

File: templates/base.html.twig (hunk 2 above)
Comment (dreadnip) [reply templates/base.html.twig]: e0f34b9

**Review by tijsverkoyen — APPROVED** [review]

**Pull request merged**